### ISSUE TITLE
bug - in man page, verbatim printf("string") becomes printf('string')

### ISSUE THREAD

**Issue by duncan-roe** [doxygen/doxygen#11271]
This has been around since doxygen 1.9.3 - I just never noticed it before.
**git bisect** identifies 6c5b9295509e as culprit. I've made a patch, will send PR soon.
Will send test case soon.

**Comment by duncan-roe** [comment]
Have discovered double-quote becoming single in regular text as well. Investigating further

**Comment by duncan-roe** [comment]
The damage is done in `ManDocVisitor::filter()` defined at line **987** of file **src/mandocvisitor.cpp**. The simplest fix is to delete line 999 `case '"':  c = '\''; // fall through`. You hardly need a PR for that.
But it seemed to me there must be a reason for having line 999. So originally, when I thought it was only a problem in verbatim code, I only changed line 230 `filter(s.text());` to be `manfilter(s.text());` where `manfilter()` was a clone of `filter()` with line 999 removed.
Then when I found wrong quoting was more widespread, I added a `bool fixquotes` argument to `filter()` and guessed that the `operator()` calls would want it on and the rest wouldn't. That turned out to be wrong, so I commented out
```
case '"':  if(fixquotes)
             c = '\''; // fall through
```
I'll send that as a PR in case it might save you a few minutes' work. The guess at line 230 is right and you'll have to decide about the rest.
In the meantime I'll upload the test case. I guess it's significant that it uses javadoc-style comments.

**Comment by duncan-roe** [comment]
[reproducer3.tar.gz](https://github.com/user-attachments/files/18121328/reproducer3.tar.gz)

**Comment by duncan-roe** [comment]
Simply **./run.sh** to run doxygen and see the man page

**Comment by duncan-roe** [comment]
I tried to send a PR but one of the tests failed, when trying to download Ghostscript from chocolatey???
Nothing I can do about that so will try sending it again.

**Comment by duncan-roe** [comment]
PR sent

**Comment by duncan-roe** [comment]
Updated PR sent

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by duncan-roe** [comment]
I built and installed Doxygen1.13.0 and can confirm #11271 is fixed
